fec/lib/test_similar_names.py

```diff
@@ -0,0 +1,21 @@
+import re
+from unittest import TestCase
+from fec.lib.similar_names import regex_name
+
+
+class SimilarNamesTest(TestCase):
+    def setUp(self):
+        self.nickname = 'SMITH, JUDY MD.'
+        self.regex = re.compile(regex_name(self.nickname))
+
+    def test_nickname_matches_given(self):
+        self.assertIsNotNone(self.regex.match('SMITH MD., JUDITH R'))
```

Comment:
Now that `regex_name` is extracted into a pure function module, the tests for it don't have to set up a whole `Contributor`, or even interact with the database at all.

Reply:
nice!